test/images/serve-hostname/VERSION

```diff
@@ -1 +1 @@
-1.0
+1.5
```

Comment:
should be `1.1` and use following instructions to push next version to `kubernetes-e2e-test-images` repo

```shell
# cd test/images
# make all-push WHAT=serve-hostname
```